src/auth/auth.types.ts

```diff
@@ -1,5 +1,27 @@
+import { FieldHookConfig } from 'formik';
+
 export interface IHandleQueryResult {
   isSuccess: boolean;
   errorMsg: string;
   errorCode: number;
 }
+
+export interface MyTextInputProps {
+  label: string;
+  name: string;
+  [key: string]: string | FieldHookConfig<any>;
+}
+
+export interface MyCheckboxProps {
+  name: string;
+  [key: string]: string | FieldHookConfig<any>;
+}
+
+export interface IAlertForm {
+  message: string;
+}
+
+export interface IButtonEl {
```

Comment:
IButtonEl does not seem to belong to `auth.types`. It should belong to other types, like `component.types.ts` and should not be in the auth folder. Same for other UI types in this file. This file should only contain auth-related stuff

Reply:
done